docs/2manifoldLearning.md

```diff
@@ -0,0 +1,46 @@
+# 2-Manifold Learning example
+
+![2-Manifold Learning example Image](https://topology-tool-kit.github.io/img/gallery/2manifoldLearning.jpeg)
+
+
+## Pipeline description
+
+This example first loads a point cloud from disk. 
+
+In a pre-processing, the data is converted to a format understandable by Paraview using the TableToPoints filter. Gaussian Resampling is applied to the data (left view in the above screenshot). This filter has the effect of injecting input points to a structured data. For each injection, each point will "splat", or distribute values to nearby vertices.
+
+Then, the [PersistenceDiagram](https://topology-tool-kit.github.io/doc/html/classttkPersistenceDiagram.html) is computed and thresholds are applied based on persistence to maintain only the most persistent features. This results in a simplified persistence diagram.
+
+The simplified persistence diagram is then used as a constraint for the [TopologicalSimplification](https://topology-tool-kit.github.io/doc/html/classttkTopologicalSimplification.html) of the input scalar data.
+
+This simplified data is then used as the input of the computation of [MorseSmaleComplex](https://topology-tool-kit.github.io/doc/html/classttk_1_1MorseSmaleComplex.html) (right view, above screenshot). This complex is composed of elements of 3 dimensions: dimension 0, which corresponds to the critical points of the Morse-Smale Complex, dimension 1, which corresponds to its edges (in grey in the screenshot) and dimension 2, which corresponds to its surfaces. The "S" shape made by the point cloud is outlined by the 1 and 2 dimension elements of the Morse-Smale Complex.
```

Comment:
3 dimensions -› 4 dimensions 

in this case, there are:
- vertices
- edges
- faces (selected in this pipeline)
- also pieces of volume (which can be extracted from the `Segmentation` output)

Reply:
"The "S" shape made by the point cloud is outlined by the 1 and 2 dimension elements of the Morse-Smale Complex."
-›
"The "S" shape made by the point cloud is outlined by the maximal 1 and 2 dimension elements of the Morse-Smale Complex."

(again, we carefully select the 1 separartrices attached to maxima here, as well as surfaces separating density voids. overall this are 1D and 2D elements in the volume that are locally maximizing the input scalar field. this is what we want. we want the output mesh to go exactly where the input point cloud is the densest)